src/js/helpers.js

```diff
@@ -494,7 +494,9 @@ ${enterpriseSettings}`;
   }
   const debInstallationCode = `wget -q -O- https://get.mender.io/${
     isPreRelease && window.location.hostname.includes('staging') ? 'staging' : ''
-  } | sudo bash -s`;
+  } | sudo bash -s${
+    isPreRelease && window.location.hostname.includes('staging') ? ' -- --demo' : ''
+  }`;
```

Comment:
```suggestion
  } | sudo bash -s${isPreRelease && window.location.hostname.includes('staging') ? ' -- --demo' : ''}`;
```
should make the linter happy (we have a prettier config with a 160 col line width 😉)